src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeRemover.vb

```diff
@@ -199,6 +227,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax
                 Return list
             End Function
```

Comment:
Ok.  I looked into this, and we definitely can't share the entire method.  First, it accesses a lot of instance state.  And we can't make the REmover classes shared either (because htey need to respectively subclass either CSharpSyntaxRewriter and VisualBasicSyntaxRewriter).  

However, i can try to share some smaller pieces of logic.